modules/cdp_engine/src/lib.rs

```diff
@@ -713,9 +713,8 @@ impl<T: Trait> RiskManager<T::AccountId, CurrencyId, Balance, Balance> for Modul
 		collateral_balance: Balance,
 		debit_balance: Balance,
 	) -> DispatchResult {
-		let debit_value = Self::get_debit_value(currency_id, debit_balance);
-
-		if !debit_value.is_zero() {
+		if !debit_balance.is_zero() {
```

Comment:
fix bug maybe lead tiny debit balance bypass MininumDebitBalance check due to caculation loss